app/src/protyle/render/abcRender.ts

```diff
@@ -3,6 +3,30 @@ import {Constants} from "../../constants";
 import {genIconHTML} from "./util";
 import {hasClosestByClassName} from "../util/hasClosest";
 
+const ABCJS_PARAMS_KEY = "%%abcjsParams:";
```

Comment:
Would it be better to use `%%params {responsive: "resize"}`?

Reply:
That sounds good to me. I'll update it to use that as well as `looseJsonParse` that you mentioned below.

Reply:
I updated the code. Let me know what you think.